EVMimports.js

```diff
@@ -59,6 +59,7 @@ module.exports = class Interface {
       'getBlockGasLimit',
       'log',
       'create',
+      '_call',
```

Comment:
I don't think this will make `call` available to contracts. is there some code somewhere removing the underscore prefix?